README.md

```diff
@@ -1,55 +1,82 @@
+- [Elasticsearch](http://www.elasticsearch.org):
+  Used for full-text search capabilities and content indexing.
+- [Node](http://nodejs.org) and NPM (Node Project Manager):
+  Used for downloading and managing front-end dependencies and assets.
 
-## Contributing
+## Installation
 
-We welcome your feedback and contributions.
-
-- [Find out about contributing](https://cfpb.github.io/capital-framework/contributing/)
-- [File a bug](https://github.com/cfpb/cfgov-refresh/issues/new?body=%23%23%20URL%0D%0D%0D%23%23%20Actual%20Behavior%0D%0D%0D%23%23%20Expected%20Behavior%0D%0D%0D%23%23%20Steps%20to%20Reproduce%0D%0D%0D%23%23%20Screenshot&labels=bug)
+#### 1. Back-end setup
 
+Follow the [Sheer installation instructions](https://github.com/cfpb/sheer#installation)
+to get sheer installed.
+**NOTE:** We suggest creating a virtualenv variable specific to this project,
+such as `cfgov-refresh` instead of `sheer` used in the Sheer installation instructions:
 
-## Getting started
+```bash
+$ mkvirtualenv cfgov-refresh
+```
 
-### Requirements
+Install the following dependencies into your virtual environment.
+We called ours `cfgov-refresh` (see previous step above):
 
-##### Back-end
-- [Sheer](https://github.com/cfpb/sheer)
-- [elasticsearch](http://www.elasticsearch.org/)
+```bash
+workon cfgov-refresh
 
-##### Front-end Build Tools
-- [Node](http://nodejs.org/) and NPM
+pip install git+git://github.com/dpford/flask-govdelivery
+pip install git+git://github.com/rosskarchner/govdelivery
+```
 
-### Back end setup
+#### 2. Front-end setup
 
-Set up [Sheer](https://github.com/cfpb/sheer#installation),
-a Jekyll-inspired, elasticsearch-powered, CMS-less publishing tool.
+The cfgov-refresh front-end currently uses the following frameworks / tools:
 
-#### Additional setup requirements for this site
+- [Grunt](http://gruntjs.com): task management for pulling in assets,
+  linting and concatenating code, etc.
+- [Bower](http://bower.io): Package manager for front-end dependencies.
+- [Less](http://lesscss.org): CSS pre-processor.
+- [Capital Framework](https://cfpb.github.io/capital-framework/getting-started):
+  User interface pattern-library produced by the CFPB.
 
-Install these dependencies into your virtual environment (we called ours 'sheer'):
+**NOTE:** If you're new to Capital Framework, we encourage you to
+[start here](https://cfpb.github.io/capital-framework/getting-started).
 
-```sh
-workon sheer
+1. Install [node.js](http://nodejs.org) however you'd like.
+2. Install [Grunt](http://gruntjs.com) and [Bower](http://bower.io):
 
-pip install git+git://github.com/dpford/flask-govdelivery
-pip install git+git://github.com/rosskarchner/govdelivery
+```bash
+$ npm install -g grunt-cli bower
 ```
 
-_We are working on a way to get these installed automatically._
+## Configuration
 
-And ask someone for the values to set the following environment variables:
+The project needs a number of environment variables.
```

Comment:
I'm unclear if they are optional or required because I haven't been able to test them without yet. We'll need to verify this. Ideally the project can be run without the need for these so someone external could in theory install and run this to change CSS etc. without the need for our proprietary backend components. ... Otherwise, what's the point of having this repo public.